org.osgi.test.cases.typedevent/bnd.bnd

```diff
@@ -0,0 +1,23 @@
+# Set javac settings from JDT prefs
+-include: ${build}/eclipse/jdt.bnd, ${build}/osgi.tck.junit-platform.bnd, ${build}/cmpn.bnd
+
+-conditionalpackage					= org.osgi.test.support.*
+-privatepackage						=  \
+	${p}.*
+Import-Package: ${-signaturetest}, *
+
+-buildpath: \
+	org.osgi.test.support;version=project,\
+	org.osgi.service.typedevent;version=latest,\
+	org.osgi.framework,\
```

Comment:
For both framework and dto, you need to specify an actual version number and also `maven-scope=provided` since they come from the framework at runtime.

```
    org.osgi.framework;maven-scope=provided;version=1.8,\
    org.osgi.dto;maven-scope=provided;version=1.0
```